### PR TITLE
Settings

### PR DESCRIPTION
**Pull request by zoilak**
added setting dropdown, i have this annoying little arrow that I am still trying to figure out how to get rid off

### CODE REVIEW

File: packages/react-frontend/src/App.jsx [hunk 1]
```diff
@@ -1,139 +1,120 @@
...
+          <Route path="/signup" element={<SignUp handleSubmit={signupUser} buttonLabel="Sign Up" />} />
+          <Route path="/profile" element={<Profile />} />
+          <Route path="/deleteAccount" element={<DeleteAccount />} />
+          <Route path="/logout" element={<Logout />} />
+        </Routes>
+      </BrowserRouter>
+      <p>{message}</p>
+    </div>
+  );
+}
```
Comment (epears04) [inline packages/react-frontend/src/App.jsx:118]: @zoilak Do you plan to have a separate page for deleting account and logout? I thought the original plan was to add a pop up.

**Review by epears04 — COMMENTED** [review]
Looks pretty good so far. Just need clarification on the plan for login and delete account pages.

File: packages/react-frontend/src/App.jsx (hunk 1 above)
Comment (zoilak) [reply packages/react-frontend/src/App.jsx:118]: Yeah I just have those as placeholders as of right now, I do think I might make them got back to the home page with a pop up.

File: packages/react-frontend/src/components/Navbar.jsx [hunk 2]
```diff
@@ -33,15 +33,32 @@ export default function Navbar() {
...
+            id="dropdown-basic"
+            style={{
+              border: "none",          
+              background: "transparent", 
+              padding: 0,            
+              position: "relative",    
+            }}
+          >
+            <img
+              src="./../../public/gear-fill.svg"
```
[the remark this answers is not in the document]
Comment (zoilak) [reply packages/react-frontend/src/components/Navbar.jsx:48]: sounds good, I'll make that change

**Pull request closed without merging**